contentcuration/contentcuration/frontend/channelEdit/components/AssessmentEditor/AssessmentEditor.vue

```diff
@@ -333,6 +334,18 @@
         this.$analytics.trackAction('exercise_editor', 'Add', {
           eventLabel: 'Question',
         });
+        this.$nextTick(() => {
+          if (this.$refs['questionCardRef'].length >= 1) {
```

Comment:
To avoid some redundancy, `this.$refs['questionCardRef']` can be defined as a constant before the if statement and reused within. 

Also consider using optional chaining `(?.)` to prevent potential errors from null values/handle them safely, for example:

if (`questionCards?.length >=  1`)

To echo @vkWeb you did a great job with this workaround!